bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Journal.java

```diff
@@ -511,9 +511,9 @@ public void run() {
             boolean shouldForceWrite = true;
             int numReqInLastForceWrite = 0;
             long busyStartTime = System.nanoTime();
+            boolean forceWriteMarkerSent = false;
```

Comment:
Are you sure this is impacting the performance? Looks like the value is to make sure the marker enqueue the forcewriterequest successfully.